crates/cluster_agent/src/main.rs

```diff
@@ -10,24 +13,51 @@ use logrecords::LogRecords;
 
 #[tokio::main]
 async fn main() -> eyre::Result<()> {
-    let (agent_health_reporter, agent_health_service) = tonic_health::server::health_reporter();
-
-    agent_health_reporter
-        .set_serving::<LogMetadataServiceServer<LogMetadata>>()
-        .await;
+    let (_, agent_health_service) = tonic_health::server::health_reporter();
 
     let reflection_service = tonic_reflection::server::Builder::configure()
         .register_encoded_file_descriptor_set(FILE_DESCRIPTOR_SET)
         .build_v1()?;
 
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::WARN)
+        .init();
+
+    let (term_tx, _term_rx) = broadcast::channel(5);
```

Comment:
Do we need to buffer the broadcast channel?

Reply:
Probably not, I picked a value that I considered safe at the time. I'll test with 1 and update

Reply:
Sounds good, mostly I'm just curious because in Go I don't use channel buffering unless it's necessary because it can hide bugs.